src/platform.ts

```diff
@@ -453,6 +461,35 @@ export class BondPlatform implements DynamicPlatformPlugin {
     Observer.add(this, fireplace.getCharacteristic(this.Characteristic.On), get, set);
   }
 
+  // Shades
+
+  private setupShadesObserver(bond: Bond, device: Device, shades?: Service) {
+    if (shades === undefined) {
+      return;
+    }
+
+    function getPosition(): Promise<CharacteristicValue> {
+      return bond.api.getState(device.id).then(state => {
+        // Always return either 1 or 100
+        return state.open === 1 ? 100 : 1;
```

Comment:
I think this should be 100 or 0?

Reply:
yup, you're right https://developers.homebridge.io/#/characteristic/CurrentPosition